src/components/SideNavbar/index.js

```diff
@@ -14,12 +16,28 @@ const Nav = styled.div`
   display: flex;
   flex-direction: column;
   align-items: center;
+
+  @media screen and (max-width: ${(props) => props.theme.deviceSizes.mobileM}) {
+    width: 100%;
+    height: 74px;
+    margin: 0 0 0 0;
+    padding: 0;
+    background-color: #fff;
+    display: flex;
+    flex-direction: row;
+    position: fixed;
+    bottom: 0;
+  }
 `;
 
 const NavContainer = styled.div`
   width: 350px;
   height: 1110px;
   display: flex;
+  @media screen and (max-width: ${(props) => props.theme.deviceSizes.mobileM}) {
+    display: flex;
+    width: 0;

```

Comment:
```suggestion
    width: 0;
    height: 74px;
```

여기에서 PC화면 기준 1110px로 지정 해두셔서 모바일 화면에서 밑에 큰 공백이 생깁니다
모바일 화면 height 값도 주셔야 할거같습니다

Reply:
이걸 놓쳤었군요 수정하겠습니다.